README.md

```diff
@@ -1 +1,1181 @@
+
+- [What you'll build](#what-you’ll-build)
+- [Prerequisites](#prerequisites)
+- [Implementation](#implementation)
+- [Testing](#testing)
+- [Configuration with Honeycomb](#testing-with-honeycomb)
+     - [Traces](#views-of-traces)
+     - [Metrics](#metrics)
+
+## What you’ll build
+To perform this integration with Honeycomb,  a real world use case of a very simple student management system is used.
+This system will illustrate the manipulation of student details in a school/college management system. The administrator will be able to perform the following actions in this service.
+
+    - Add a student's details to the system.
+    - List down all the students' details who are registered in the system.
+    - Delete a student's details from the system by providing student ID.
+    - Generate a mock error (for observability purposes).
+    - Get a student's marks list by providing student ID.
+
+![Honeycomb](images/ballerina-honeycomb.svg "Ballerina-Honeycomb")
+
+- **Make Requests** : To perform actions on student  management service, a console based client program has been written in Ballerina for your ease of making requests.
+
+## Prerequisites
+ 
+- [Ballerina Distribution](https://ballerina.io/learn/getting-started/)
+- [Docker](https://docs.docker.com/engine/installation/)
+- [MySQL](https://github.com/ballerina-guides/ballerina-honeycomb/blob/master/resources/testdb.sql)
+- A Text Editor or an IDE 
+
+## Implementation
+
+> If you want to skip the basics, you can download the GitHub repo and continue from the [Testing](#testing) section.
+
+### Implementing the database
+ - Start MySQL server in your local machine.
+ - Create a database with name `testdb` in your MySQL localhost. If you want to skip the database implementation, then directly import the [testdb.sql](https://github.com/ballerina-guides/ballerina-honeycomb/blob/master/resources/testdb.sql) file into your localhost. You can find it in the Github repo.
+### Create the project structure
+        
+ For the purpose of this guide, let's use the following package structure.
+        
+    
+    ballerina-honeycomb
+           └── guide
+                ├── students
+                │   ├── student_management_service.bal
+                │   ├── student_marks_management_service.bal
+                ├── client_service
+                |         └── client_main.bal
+                └── ballerina.conf
+        
+
+- Create the above directories in your local machine, along with the empty `.bal` files.
+
+- Add the following lines in your [ballerina.conf](https://github.com/ballerina-guides/ballerina-honeycomb/blob/master/ballerina.conf) to send the service traces to Honeycomb in Zipkin format using Opentracing.
+
+```ballerina
+[b7a.observability.tracing]
+enabled=true
+name="zipkin"
+
+[b7a.observability.tracing.zipkin]
+reporter.hostname="localhost"
+reporter.port=9411
+
+# Send the spans in V1 format as honeycomb-opentracing-proxy supports on V1
```

Comment:
This seems a bit complex to follow. Can we give more explanation on what needs to be done here and why we are doing it?